src/overlays/actors/ovl_En_Ishi/z_en_ishi.c

```diff
@@ -9,7 +18,32 @@ void EnIshi_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnIshi_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnIshi_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void EnIshi_SetupWait(EnIshi* this);
+void EnIshi_Wait(EnIshi* this, GlobalContext* globalCtx);
+void EnIshi_SetupLiftedUp(EnIshi* this);
+void EnIshi_LiftedUp(EnIshi* this, GlobalContext* globalCtx);
+void EnIshi_SetupFly(EnIshi* this);
+void EnIshi_Fly(EnIshi* this, GlobalContext* globalCtx);
+void EnIshi_SpawnFragmentsSmall(EnIshi* this, GlobalContext* globalCtx);
+void EnIshi_SpawnFragmentsLarge(EnIshi* this, GlobalContext* globalCtx);
+void EnIshi_SpawnDustSmall(EnIshi* this, GlobalContext* globalCtx);
+void EnIshi_SpawnDustLarge(EnIshi* this, GlobalContext* globalCtx);
+
+extern Gfx D_0500A3B8[]; // Large gray rock
+extern Gfx D_0500A5E8[]; // Large gray rock fragments
+extern Gfx D_0500A880[]; // Small gray rock
+
+typedef void (*EnIshiEffectSpawnFunc)(struct EnIshi*, GlobalContext*);
+typedef void (*EnIshiDrawFunc)(struct EnIshi*, GlobalContext*);
+
+typedef enum {
+    /* 0x00 */ ROCK_SMALL,
+    /* 0x01 */ ROCK_LARGE
+} EnIshiType;
```

Comment:
this should be in the header so that its public, other things will use this for params when spawning it